### DISCUSSION TITLE
Converting GGUF to HF Safetensors

### DISCUSSION THREAD

**Discussion by PCanavelli** [ggml-org/llama.cpp#9410] [Q&A]
Hi everyone

I understand that this question is entirely backwards, but I have to deal with a peculiar use case: I have a GGUF Flux checkpoint I would like to try in a Diffusers pipeline.

Is there any way to use llama.ccp to perform the reverse conversion from GGFU to safetensors?

Cheers

**Comment by francisfelici** [comment]


**Reply by purinnohito** [reply]
If you still need it, a script I made to fulfill a personal demand
https://github.com/purinnohito/gguf_to_safetensors
Please refer to it as there is
What we are doing is almost
https://github.com/ggerganov/llama.cpp/blob/master/gguf-py/gguf/quants.py#L67
and all it is doing is calling
*Don't expect any support as I am not good at python
